libs/gretty/src/main/groovy/org/akhikhl/gretty/AppStartTask.groovy

```diff
@@ -25,6 +27,144 @@ class AppStartTask extends StartBaseTask {
   @Delegate
   private WebAppConfig webAppConfig = new WebAppConfig()
 
+  @Input
+  @Optional
+  getContextPath() {
```

Comment:
A general comment: The return type is missing for all these methods. I think you need to add the return type, so they actually override the method from the delegate. And it would be clearer as well I think.

Reply:
Good call. That's reasonable, will do.